tokio-console/src/view/task.rs

```diff
@@ -332,6 +353,21 @@ impl Details {
         text.extend(percentiles);
         text
     }
+
+    /// Render the consequences as an ascii tree.
+    fn make_stack_widget(&self, _styles: &view::Styles, state: &State) -> Text<'static> {
+        // XXX(jswrenn): also implement an ASCII-only rendering mode
```

Comment:
i think it's fine to just not display the stack in ASCII-only mode, if it's difficult to implement an ASCII-only rendering for this. we already don't display the poll duration histogram in ASCII-only mode...